src/hooks/useLocalhostQuickstart.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+
+import { getQuickstartFilesFromLocal } from '../utils/preview/fetchHelpers';
+import { parseQuickstartFiles } from '../utils/preview/parseHelpers';
+import { navigate } from 'gatsby';
+
+const usePullRequestQuickstart = (location) => {
+  const [contentFiles, setContentFiles] = useState([]);
```

Comment:
```suggestion
  const [quickstart, setQuickstart] = useState([]);
```

I made the same comment in #192 - we might want to consider changing this to hold the state of the quickstart, since we dont need to worry about the files once they're parsed into a quickstart